--- a/src/contest_util.c
+++ b/src/contest_util.c
@@ -1415,57 +1415,57 @@
     y = 1;
     if (gLinkContestFlags & LINK_CONTEST_FLAG_IS_LINK)
     {
-        CopyToBgTilemapBufferRect(2, gContestResultsTitle_Link_Tilemap, 5, 1, 5, 2);
-        x = 10;
+        CopyToBgTilemapBufferRect(2, gContestResultsTitle_Link_Tilemap, 5, 1, GAME_LANGUAGE == LANGUAGE_SPANISH ? 10 : 5, 2);
+        x = GAME_LANGUAGE == LANGUAGE_SPANISH ? 20 : 10;
     }
     else if (gSpecialVar_ContestRank == CONTEST_RANK_NORMAL)
     {
         CopyToBgTilemapBufferRect(2, gContestResultsTitle_Normal_Tilemap, 5, 1, 10, 2);
-        x = 15;
+        x = GAME_LANGUAGE == LANGUAGE_SPANISH ? 20 : 15;
     }
     else if (gSpecialVar_ContestRank == CONTEST_RANK_SUPER)
     {
         CopyToBgTilemapBufferRect(2, gContestResultsTitle_Super_Tilemap, 5, 1, 10, 2);
-        x = 15;
+        x = GAME_LANGUAGE == LANGUAGE_SPANISH ? 20 : 15;
     }
     else if (gSpecialVar_ContestRank == CONTEST_RANK_HYPER)
     {
         CopyToBgTilemapBufferRect(2, gContestResultsTitle_Hyper_Tilemap, 5, 1, 10, 2);
-        x = 15;
+        x = GAME_LANGUAGE == LANGUAGE_SPANISH ? 20 : 15;
     }
     else // CONTEST_RANK_MASTER
     {
         CopyToBgTilemapBufferRect(2, gContestResultsTitle_Master_Tilemap, 5, 1, 10, 2);
-        x = 15;
+        x = GAME_LANGUAGE == LANGUAGE_SPANISH ? 20 : 15;
     }

     if (gSpecialVar_ContestCategory == CONTEST_CATEGORY_COOL)
     {
         palette = 0;
-        CopyToBgTilemapBufferRect(2, gContestResultsTitle_Cool_Tilemap, x, y, 5, 2);
+        CopyToBgTilemapBufferRect(2, gContestResultsTitle_Cool_Tilemap, x, y, GAME_LANGUAGE == LANGUAGE_SPANISH ? 6 : 5, 2);
     }
     else if (gSpecialVar_ContestCategory == CONTEST_CATEGORY_BEAUTY)
     {
         palette = 1;
-        CopyToBgTilemapBufferRect(2, gContestResultsTitle_Beauty_Tilemap, x, y, 5, 2);
+        CopyToBgTilemapBufferRect(2, gContestResultsTitle_Beauty_Tilemap, x, y, GAME_LANGUAGE == LANGUAGE_SPANISH ? 6 : 5, 2);
     }
     else if (gSpecialVar_ContestCategory == CONTEST_CATEGORY_CUTE)
     {
         palette = 2;
-        CopyToBgTilemapBufferRect(2, gContestResultsTitle_Cute_Tilemap, x, y, 5, 2);
+        CopyToBgTilemapBufferRect(2, gContestResultsTitle_Cute_Tilemap, x, y, GAME_LANGUAGE == LANGUAGE_SPANISH ? 6 : 5, 2);
     }
     else if (gSpecialVar_ContestCategory == CONTEST_CATEGORY_SMART)
     {
         palette = 3;
-        CopyToBgTilemapBufferRect(2, gContestResultsTitle_Smart_Tilemap, x, y, 5, 2);
+        CopyToBgTilemapBufferRect(2, gContestResultsTitle_Smart_Tilemap, x, y, GAME_LANGUAGE == LANGUAGE_SPANISH ? 6 : 5, 2);
     }
     else // CONTEST_CATEGORY_TOUGH
     {
         palette = 4;
-        CopyToBgTilemapBufferRect(2, gContestResultsTitle_Tough_Tilemap, x, y, 5, 2);
+        CopyToBgTilemapBufferRect(2, gContestResultsTitle_Tough_Tilemap, x, y, GAME_LANGUAGE == LANGUAGE_SPANISH ? 6 : 5, 2);
     }

     x += 5;
-    CopyToBgTilemapBufferRect(2, gContestResultsTitle_Tilemap, x, y, 6, 2);
+    CopyToBgTilemapBufferRect(2, gContestResultsTitle_Tilemap, GAME_LANGUAGE == LANGUAGE_SPANISH ? 15 : x, y, GAME_LANGUAGE == LANGUAGE_SPANISH ? 5 : 6, 2);
     CopyToBgTilemapBufferRect_ChangePalette(2, sContestResults->tilemapBuffers[2], 0, 0, 32, 4, palette);
 }
