--- a/src/credits.c
+++ b/src/credits.c
@@ -119,4 +119,8 @@
 static void DeterminePokemonToShow(void);

+#if GAME_LANGUAGE == LANGUAGE_SPANISH
+static const u32 sSpanishTheEnd_Tilemap[] = INCBIN_U32("graphics/credits/spanish_the_end.bin.lz");
+#endif
+
 static const u8 sTheEnd_LetterMap_T[] =
 {
@@ -1329,4 +1333,7 @@
 static void DrawTheEnd(u16 offset, u16 palette)
 {
+#if GAME_LANGUAGE == LANGUAGE_SPANISH
+    LZ77UnCompVram(sSpanishTheEnd_Tilemap, (void *)(VRAM + offset));
+#else
     u16 pos;
     u16 baseTile = (palette / 16) << 12;
@@ -1341,3 +1348,4 @@
     DrawLetterMapTiles(sTheEnd_LetterMap_N, 20, 7, offset, palette);
     DrawLetterMapTiles(sTheEnd_LetterMap_D, 24, 7, offset, palette);
+#endif
 }
